framework/src/main/java/org/checkerframework/framework/type/poly/AbstractQualifierPolymorphism.java

```diff
@@ -32,10 +32,15 @@
 import org.checkerframework.javacutil.TreeUtils;
 
 /**
- * Implements framework support for qualifier polymorphism.
+ * Implements framework support for qualifier polymorphism. Works for a particular type system,
```

Comment:
I don't think the extra parts add anything useful.

Reply:
Unless you object, I'm going to leave it there because it is helpful to people who aren't you. :-)